src/repository/courtRepository.ts

```diff
@@ -0,0 +1,19 @@
+import { Athlete } from "../model/athleteModel.js";
+
+const courtPlayers: Athlete[] = [];
```

Comment:
**Question:** Does it make sense to tell that when an athlete joins the court it become a player?

Reply:
For me it does